estate/tests/test_estate_property.py

```diff
@@ -0,0 +1,105 @@
+from odoo.tests.common import TransactionCase
+from odoo.exceptions import UserError
+from odoo.tests import Form, tagged
+
+
+@tagged('post_install', '-at_install')
+class EstateTestCase(TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # add env on cls and many other things
+        super().setUpClass()
+
+        cls.properties = cls.env['estate.property'].create([{'name': 'test_house'}])
```

Comment:
Plural is misleading
```suggestion
        cls.property = cls.env['estate.property'].create([{'name': 'test_house'}])
```